packages/jest-config/src/reporterValidationErrors.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+const {ValidationError} = require('jest-validate');
+const {DOCUMENTATION_NOTE, BULLET} = require('./utils');
+
+const chalk = require('chalk');
+const {getType} = require('jest-matcher-utils');
+
+const validReporterTypes = ['array', 'string'];
+const ERROR = `${BULLET} Reporter Validation Error`;
+
+/**
+ * Reporter Vaidation Error is thrown if the given arguments
+ * within the reporter are not valid
+ * 
+ * This is a highly specific reporter error and in the future will be
+ * merged with jest-validate. Till then, we can make use of it. It works
+ * and that's what counts most at this time
+ * 
+ * @param {Number} reporterIndex specific index at which reporter is present
+ * @param {any} reporterValue value of the reporter, anything output by the user
+ * 
+ * @returns {ValidationError} validation error which can be thrown if needed
+ * 
+ */
+function createReporterError(
+    reporterIndex:number, reporterValue: any
+) : ValidationError {
+    const errorMessage = (
+        `\tReporter at index ${reporterIndex} must be of type:\n` +
```

Comment:
@thymikee Confused too about this one, I think I can remove `\t` as you are right, this hasn't been used previously anywhere in the code. I think I can replace them with two spaces for each \t